.github/CODE_STYLE.md

```diff
@@ -135,7 +141,7 @@ from a tegel component.
 The events are named according to our naming convention: 'tds' + event. For a click event this would result in the event being called `tdsClick`. 
 
 ### Internal events
-Some of the component are using event to communicate with its parent/child. These events are not recommended to use in any way since
+Some of the components are using events to communicate with its parent/child. These events are not recommended to use in any way since
```

Comment:
```suggestion
Some of our components are using events to communicate with its parent/child. These events are not recommended to use in any way since
```